lib/src/logger.dart

```diff
@@ -21,7 +23,12 @@ abstract class Logger {
 
   /// Creates a logger that prints warnings to standard error, with terminal
   /// colors if [color] is `true` (default `false`).
-  const factory Logger.stderr({bool color}) = StderrLogger;
+  factory Logger.stderr({bool color = false}) {
```

Comment:
Unfortunately, converting a factory constructor from const to non-const is a breaking change in Dart.  This means that instead of a separate `BrowserLogger` class, you'll need to use `if (isBrowser)` in `StderrLogger` itself. Sorry I didn't catch this earlier.